tests/integration_test/action_service_test.py

```
@@ -12911,28 +13108,23 @@ def mock_completion_for_answer(*args, **kwargs):
     from uuid6 import uuid7
 
+    llm_type = "openai"
     action_name = "kairon_faq_action"
     bot = "5u80fd0a56c908ca10d35d2s"
     user = "udit.pandey"
     value = "keyvalue"
     user_msg = "What is the name of prompt?"
     bot_content = "Python is a high-level, general-purpose programming language. Its design philosophy emphasizes code readability with the use of significant indentation. Python is dynamically typed and garbage-collected."
     generated_text = "Python is dynamically typed, garbage-collected, high level, general purpose programming."
+    hyperparameters = Utility.get_default_llm_hyperparameters()
+    test_prompt_action_response_action_slot_prompt
```

Ruff

Found useless expression. Either assign it to a variable or remove it.

(B018)